src/eos/twoPhase.cpp

```diff
@@ -26,14 +26,14 @@ static inline void SimpleGasGasDecode(PetscInt dim, ablate::eos::TwoPhase::Decod
         rho2 = rho;
         e2 = e;
         p = (gamma2 - 1) * rho2 * e2;
-        T = e2 * cv2;
+        T = e2 / cv2;
```

Comment:
Were these wrong?  Can you add some tests that would have caught this bug?

Reply:
yes, I can add some test cases for this